source/ConditionSet.h

```diff
@@ -96,6 +96,7 @@ class ConditionSet {
 
 	// Check if the given condition values satisfy this set of expressions.
 	bool Test(const ConditionsStore &conditions) const;
```

Comment:
Should we remove this one, when we have the other one?

Reply:
I'll remove this one in a follow-up PR where I update all existing ConditionSets. Or I could split the ConditionSet changes out of this PR and do it in a parallel one right now.

Reply:
#11350